src/Helpers/misc.php

```diff
@@ -91,7 +91,7 @@ function split_data(?string $values): array
 function a2b_round($number, $PRECISION = 6): float
 {
 
```

Comment:
Better approach would probably be to properly type `$number` as an int, and find out where the bad uses are, but that sounds like a lot of work so this is probably the better approach